packages/web-app-files/src/components/SideBar/Shares/Links/DetailsAndEdit.vue

```diff
@@ -237,14 +240,14 @@ export default defineComponent({
       return LinkShareRoles.getByBitmask(this.link.permissions, this.isFolderShare)
     },
     currentLinkRoleDescription() {
-      return this.currentLinkRole.description(false)
+      return this.currentLinkRole?.description(false) || ''
     },
 
     currentLinkRoleLabel() {
-      if (this.currentLinkRole.longLabel !== '') {
+      if (this.currentLinkRole?.longLabel !== '') {
         return this.$gettext(this.currentLinkRole.longLabel)
       } else {
```

Comment:
no else needed, but nitpick otherwise fine :)

Reply:
True, I don't touch code usually while fixing but the else triggers me as well, lol